js/tabulus.js

```diff
@@ -13,7 +13,19 @@ function Tabulus() {
     ** Main Function Object
     */
     function my(sel) {
-      if(!container) setup(sel);
+        if(!container) setup(sel);
+        d3.selectAll("a[data-toggle='modal']")
+            .each(function() {
+                var self = d3.select(this)
+                  , name = self.attr("data-name")
+                ;
+                self
```

Comment:
I'm not understanding the benefit of these changes here. What impact do these changes have? It almost seems simpler to keep this bit in Jekyll, as we still are scaffolding out these buttons with Jekyll anyway.

Also it looks like `href` and `data-target` are being set twice, once in Jekyll and again in JavaScript? Making these be set only in one place is the only change I'd want to see before merging this PR.

My gut feeling here, relating to workflow, is that it might be easier to keep changes that relate to different features in different PRs. e.g. this change could go in a separate PR from the legend-related changes. The legend changes are great, but these changes to the modals seem not quite finished.